admin/acceptance_tests/layers_wms_test.py

```diff
@@ -323,17 +353,25 @@ def test_duplicate(self, layer_wms_test_data, test_app, dbsession):
         assert layer.dimensions[2].value == form.html.select(
             'input[value=dimensions:mapping] + input + div + div')[2].findChild('input').attrs['value']
 
-        self.set_first_field_named(form, 'name', 'clone')
-        resp = form.submit('submit')
+    def _check_interfaces_are_checked(self, form, interfaces, layer):
```

Comment:
Not that this not only test the checkboxes status, it also test labels and values, so I would name it _check_interfaces